opentelemetry-sdk/src/trace/config.rs

```diff
@@ -129,9 +129,9 @@ impl Default for Config {
                     if let Some(r) = ratio {
                         Box::new(Sampler::TraceIdRatioBased(r))
                     } else {
-                        handle_error(
-                            Error::Other(String::from(
-                                "Missing or invalid OTEL_TRACES_SAMPLER_ARG value. Falling back to default: 1.0"))
+                        otel_warn!(
+                            name: "TracesSampler.TraceIdRatio.Error",
+                            message= "Missing or invalid OTEL_TRACES_SAMPLER_ARG env variable. Falling back to default: 1.0"
```

Comment:
need more user actionable message.
OTEL_TRACES_SAMPLER env variable has the value traceidratio and hence ratio should be provided by OTEL_TRACES_SAMPLER_ARG env variable which is missing/invalid. TraceIdRatioBased sampler will still be enabled, but will a fallback value of 1.0 (i.e 100% traces are accepted) for ratio.

Reply:
done